third_party/nvfuser/csrc/arith.cpp

```diff
@@ -808,6 +808,49 @@ Val* ones_like(Val* v) {
   return ones_like(v->as<TensorView>());
 }
 
+TensorView* iota(Val* length, Val* start, Val* step, DataType dtype) {
```

Comment:
This is great! In the future it may be interesting to consider generalizing this to accept a shape and a dimension, like XLA's iota op